index.js

```diff
@@ -40,24 +55,41 @@ module.exports = {
     function getBytesStored(feedId, cb) {
       /** @type {IndexPlugin} */
       const indexPlugin = ssb.db.getIndex('storageUsed')
-      ssb.db.onDrain('storageUsed', () => {
-        cb(null, indexPlugin.getBytesStored(feedId))
+      stats((err, info) => {
+        if (err) return cb(err)
+        ssb.db.onDrain('storageUsed', () => {
+          const logBytes = indexPlugin.bytesStored.get(feedId)
+          if (logBytes == null) return cb(null, 0)
```

Comment:
This line would have been good to have even before this PR. It means that if an unknown feedId is passed as input, we can safely assume that there are zero bytes used by that feed.